contracts/Governance/AccessControlManager.sol

```diff
@@ -63,11 +63,11 @@ contract AccessControlManager is AccessControl {
     /**
      * @notice Gives a function call permission to one single account
      * @dev this function can be called only from Role Admin or DEFAULT_ADMIN_ROLE
-     * 		May emit a {RoleGranted} event.
+     * 		May emit a {RoleGranted} and {PermissionGranted} events.
      * @param contractAddress address of contract for which call permissions will be granted
-     * NOTE: if contractAddress is zero address, we give the account DEFAULT_ADMIN_ROLE,
-     *      meaning that this account can access the certain function on ANY contract managed by this ACL
-     * @param functionSig signature e.g. "functionName(uint,bool)"
+     * NOTE: if contractAddress is zero address, the account can access the certain function
```

Comment:
For the automatic doc generation, I started to set it up so that `@dev` tags get written to info hint blocks. For each of these notes we could replace `NOTE:` with `@dev` and they will get placed in their own info hint blocks in the generated docs

```suggestion
     * NOTE: if contractAddress is zero address, the account can access the specified function
```

Reply:
Fixed in 6484b42